app/models.py

```diff
@@ -310,3 +312,13 @@ class PageVisits(PageVisit):
 
 class Stats(BaseModel):
     date: date
+
+
+class KioskSuggestionPayload(BaseModel):
+    product: str
+
+
+class KioskSuggestion(KioskSuggestionPayload):
+    id: str
+    member: Member
```

Comment:
Not a requirement, and only admins are allowed to view this data as it can be sensitive. However, I thought it was a nice utility if admins want to ask a person what they mean by certain suggestions, or hold users accountable for spamming or other abuse.